packages/@aws-cdk/aws-efs/lib/file-system.ts

```diff
@@ -0,0 +1,38 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import {IResource} from "@aws-cdk/core";
+
+/**
+ * Interface to implement AWS File Systems.
+ */
+export interface IFileSystem extends IResource, ec2.IConnectable {
+    /**
+     * The ID of the file system, assigned by Amazon EFS.
+     *
+     * @attribute
+     */
+    readonly fileSystemID: string;
+}
+
+/**
+ * Properties of FileSystem
+ */
+export interface FileSystemProps {
```

Comment:
Fold this into `EfsFileSystemProps` as well.

Reply:
updated.